inventory/ipv4_addresses/inventory_ipv4_addresses.cf

```diff
@@ -0,0 +1,24 @@
+bundle agent inventory_ipv4_addresses
+{
+  vars:
+    "filter_reg"
+      string => "127\.0\.0\.1",
```

Comment:
OK, makes sense. I thought you tried to filter out localhost, not just a pattern of common addresses. But I see now what you're doing, so it makes sense!